src/interface/viewport.c

```diff
@@ -1140,6 +1097,51 @@ int sub_98198C(
 }
 
 /**
+ * rct2: 68818E
+ *
+ * @param image_id (ebx)
+ * @param x (ax)
+ * @param y (cx)
+ * @param[out] paint (ebp)
+ * @return (!CF) success
+ */
+bool sub_68818E(uint32 image_id, uint8 x, uint8 y, paint_struct ** paint)
```

Comment:
This function attaches a paint struct to another paint struct. Perhaps call it `paint_setup_attach_paint_struct` or something better.